packages/storybook/src/stories/grid-sorting.stories.tsx

```diff
@@ -435,8 +435,9 @@ function sortServerRows(rows: any[], params: GridSortModelParams): Promise<any[]
       }
       const sortedCol = params.sortModel[0];
       const comparator = params.columns[0].sortComparator!;
+      const clonedRows = [...rows];
       let sortedRows = [
-        ...rows.sort((a, b) => comparator(a[sortedCol.field], b[sortedCol.field], a, b)),
+        ...clonedRows.sort((a, b) => comparator(a[sortedCol.field], b[sortedCol.field], a, b)),
```

Comment:
why double `...clonedRows`  and `const clonedRows = [...rows];` ?

Reply:
Oh yes. I removed double spreading now.